tests/travis_install.sh

```diff
@@ -42,4 +43,4 @@ fi
 # Step 3: install yt
 $PIP install -e .
```

Comment:
fyi: there is no editable install with PEP517 projects anymore [[1]](https://www.python.org/dev/peps/pep-0517/#get-requires-for-build-sdist) [[2]](https://github.com/pypa/pip/issues/7953#issuecomment-646588548)

Reply:
Oh, I didn't know that! I believe a simple fix would be to use `$PIP install .` instead, right?

Reply:
Just found out as well.

According to my understanding, PEP517 packages should be installed locally like this:
```bash
python3 -m pip wheel .
python3 -m pip install *whl
```

brave new world, I guess.

Reply:
wow. Sorry for asking the naive question, but by "PEP517 project" you mean any project with a `pyproject.toml` file, right ?

Reply:
Yep... I know...

Reply:
thanks for bringing this up, I was the person who introduced pyproject.toml to the project but wasn't aware of this :/